client/app/bundles/course/survey/pages/ResponseShow/__test__/index.test.jsx

```diff
@@ -31,6 +31,7 @@ describe('<ResponseShow />', () => {
         id: responseId,
         creator_name: 'Staff',
         submitted_at: '2099-12-31T16:00:00.000Z',
+        updated_at: '2099-12-31T16:00:00.000Z',
```

Comment:
Maybe can change this value to be different from `submitted_at`.

Would be good to also test if it's red colour when it's updated after survey deadline.

Reply:
Thanks and updated! The red colour is only currently shown on the ResponseIndex page but not in ResponseShow page so I will just leave it as it is first.